x-pack/plugins/cases/server/connectors/thehive/format.test.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Case } from '../../../common/types/domain';
+import { format } from './format';
+
+describe('TheHive formatter', () => {
+  const theCase = {
+    tags: ['tag1'],
+    severity: 'high',
+    connector: { fields: { tlp: '1' } },
+  } as Case;
+
+  it('it formats correctly', async () => {
+    const res = await format(theCase, []);
+    expect(res).toEqual({ tlp: 1, tags: ['tag1'], severity: 3 });
+  });
+
+  it('it formats correctly when fields do not exist ', async () => {
+    const invalidFields = { tags: ['tag1'], severity: 'low', connector: { fields: null } } as Case;
+    const res = await format(invalidFields, []);
+    expect(res).toEqual({ tlp: null, severity: 1, tags: ['tag1'] });
```

Comment:
Should we default the `tlp` to something if it is a required field on TheHive?

Reply:
We've set the default `TLP` to `AMBER` in the frontend, so it will always be present in the `format` function.

Reply:
Users can use the API and they can set the `tlp` to `null`. Does TheHive accepts `null` as a TLP?

Reply:
Yes, TheHive will accept it as `null` and will default set it to `AMBER`.

Reply:
Oh perfect then!